components/profile/AccountSettings.tsx

```diff
@@ -0,0 +1,174 @@
+import axios from "axios";
+import Router from "next/router";
+import React from "react";
+import useSWR, { mutate } from "swr";
+import { Avatar,Row,Col,Typography,Input } from 'antd';
+
+import ListErrors from "../common/ListErrors";
+import checkLogin from "../../lib/utils/checkLogin";
+import { SERVER_BASE_URL } from "../../lib/utils/constant";
+import storage from "../../lib/utils/storage";
+
+const SettingsForm = () => {
+  const [isLoading, setLoading] = React.useState(false);
+  const [errors, setErrors] = React.useState([]);
+  const [userInfo, setUserInfo] = React.useState({
+    image: "",
+    username: "",
+    bio: "",
+    email: "",
+    password: "",
+    github:"",
+    twitter:"",
+    website:"",
+    linkedIn:"",
+  });
+  const { Text } = Typography;
+  const textStyle={
```

Comment:
We should be using inline styles or css classes. The other pages and components use styled-components. So this page should do the same as well. The ant design components should be styled with styled components as well

Reply:
used styled components